_posts/2023-02-16-secure-cloud-foundation.md

```diff
@@ -0,0 +1,209 @@
+
+To solve the vulnerability you can read the details of the vulnerability and a best practice for prevention if
+available.
+There is also another tab that shows 3 open source projects that had the vulnerability and how they fixed it their code.
+
+### Licence scanning
+
+You can configure Snyk to scan your open source dependencies for licence issues with your dependencies,
+which can be useful for example if your company wants to avoid using dependencies with a certain licence,
+because they want to commercialize the software in the future.
+
+### Container scanning
+
+Snyk can scan your container images for vulnerabilities it can do this from docker images, Amazon ECR, Docker hub, ...
+You can set this up in your Kubernetes cluster, but currently Fargate is not supported.
+
+For scanning that is linked to a git repository you have the option
+to create a pull request to update to a fixed image version.
+
+### Infrastructure as code scanning
+
+Snyk can scan your infrastructure as code against the CIS AWS Foundations Benchmark or you can write custom policies.
+
+### Cloud scanning
+
+Snyk also has cloud scanning, but this service is still in closed beta and is not available for everyone.
+We were not able to test this product, because it requires an enterprise licence at the time of writing.
+
+### Custom policies
+
+Snyk allows you to write custom policies in Rego, but only for IaC scanning and platform policies.
+
+## The Container Detective: Trivy
+
+Trivy is an open-source cli tool provided by Aqua Security.
+
+### Container scanning
+
+Trivy can scan container images against well known vulnerabilities.
+On the [tool's homepage](https://trivy.dev/), you can enter public available docker hub images to test it out.
+
+### Dependency scanning
+
+Trivy can scan your dependencies for well known vulnerabilities.
+It has a mode that automatically discovers, declarations files for various package managers
+
+### CI/CD integration
+
+Because Trivy is a cli tool it can easily be integrates in CI/CD pipelines.
+Trivy also maintains a [GitHub action](https://github.com/aquasecurity/trivy-action) to integrate it in GitHub actions.
+But the community has created 2 additional GitHub actions.
+
+### AWS integration
+
+Trivy can be run locally to scan your AWS environment using the AWS cli.
+The default included check scans against AWS CIS 1.2.0 benchmark.
+It shows summarizes a lists of issues, and gives description of how to resolve the issue, it won't automatically fix it.
+
+### Secret scanning
+
+Trivy can scan your code for secrets, it can scan for AWS access key, GCP service account, GitHub personal access token,
+GitLab personal access token, Slack access token, etc.
+
+It can do this either on the file system or inside a container image.
+
+### Configuration issues
+
+Trivy can scan your configuration files like Dockerfiles, Kubernetes manifests, Terraform, CloudFormation, etc.
```

Comment:
If the others are all just one-line explanations, make it a list instead of separate headers. 

So either add a bit more explanation on what it does, how it does it and why or move all of them to a simple list.